src/harness/testcases/WINNF_FT_S_SSS_testcase.py

```diff
@@ -124,16 +165,143 @@ def test_WINNF_FT_S_SSS_12(self, config_filename):
 
   @winnforum_testcase
   def test_WINNF_FT_S_SSS_13(self):
-    """ Disallowed TLS method attempted during registration.
+    """ Disallowed TLS method attempted by SAS Test Harness.
 
     Checks that SAS UUT response with fatal alert message.
     """
     self.assertTlsHandshakeFailure(SAS_CERT, SAS_KEY, ssl_method=SSL.TLSv1_1_METHOD)
 
   @winnforum_testcase
   def test_WINNF_FT_S_SSS_14(self):
-    """Invalid ciphersuite presented during registration.
+    """Invalid ciphersuite presented by SAS Test Harness.
 
     Checks that SAS UUT response with fatal alert message.
     """
     self.assertTlsHandshakeFailure(SAS_CERT, SAS_KEY, ciphers='ECDHE-RSA-AES256-GCM-SHA384')
+
+  def generate_SSS_15_default_config(self, filename):
+    """Generates the WinnForum configuration for SSS_15. """
+    # Create the actual config for SAS cert/key path
+
+    config = {
+        'sasCert': self.getCertFilename("sas_inapplicable.cert"),
+        'sasKey': self.getCertFilename("sas.key")
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_SSS_15_default_config)
+  def test_WINNF_FT_S_SSS_15(self, config_filename):
+    """Certificate with inapplicable fields presented during Full Activity Dump 
+    """
+    config = loadConfig(config_filename)
+    
+    # Reset the SAS UUT 
+    self.SasReset()
+
+    # Notify SAS UUT about peer SAS
+    certificate_hash = getCertificateFingerprint(config['sasCert'])
+    self._sas_admin.InjectPeerSas({'certificateHash': certificate_hash,\
+                                     'url': SAS_TEST_HARNESS_URL })
+    try:
+       self.assertTlsHandshakeSucceed(self._sas_admin._base_url, ['AES128-GCM-SHA256'], SAS_CERT, SAS_KEY)
```

Comment:
The TLS handshake is supposed to succeed, as the 'inapplicable fields' is managed only at the application layer.
So here the test will end and succeed just after the assertTlsHandshakeSucceed, while it is supposed to do the TriggerFullActivityDumpAndWaitUntilComplete

I suggest to rewrite it as:
```python
    config = loadConfig(config_filename)
    
    # Reset the SAS UUT 
    self.SasReset()

    # Notify SAS UUT about peer SAS
    certificate_hash = getCertificateFingerprint(config['sasCert'])
    self._sas_admin.InjectPeerSas({'certificateHash': certificate_hash,\
                                     'url': SAS_TEST_HARNESS_URL })
    self.assertTlsHandshakeSucceed(self._sas_admin._base_url,
                                   ['AES128-GCM-SHA256'], SAS_CERT, SAS_KEY)
    trigger_succeed = False
    # Initiate Full Activity Dump
    try:
      self.TriggerFullActivityDumpAndWaitUntilComplete(config['sasCert'], config['sasKey'])
      trigger_succeed = True
    except AssertionError as e:
      # Check if HTTP status is 403
      self.assertEqual(e.args[0], 403)
    self.assertFalse(trigger_succeed, "Full Activity Dump is expected to fail")
```

Reply:
Thanks, Norbert. Sorry, this was my bad. got distracted by a few meetings and did it wrong. You are right, fixed now.